tests/integration_tests/modules/test_active_modules.py

```diff
@@ -0,0 +1,11 @@
+import pytest
+
+from tests.integration_tests.instances import IntegrationInstance
+from tests.integration_tests.util import get_inactive_modules, verify_clean_log
+
+
+@pytest.mark.ci
+def test_active_modules(client: IntegrationInstance):
```

Comment:
I'd want to avoid a separate integration test for this, as it's costly to spin up a new VM for something that we can bolt onto an existing test.

Reply:
Integrated into [tests/integration_tests/modules/test_ca_certs.py](https://github.com/canonical/cloud-init/pull/1609/files#diff-047fb889eb719118be33c0be34ddd062b1a6f6cc2651f1bdb0bfbd3ab6787a0cR95).